src/affine.rs

```diff
@@ -223,6 +223,28 @@ impl Affine {
         self.0[0] * self.0[3] - self.0[1] * self.0[2]
     }
 
+    /// The ["entry-wise" L<sub>1</sub> norm][matrix-norm#entry-wise], also known as
+    /// the L<sub>1,1</sub> norm.
+    ///
+    /// This norm is equivalent to considering the affine matrix as a 3x3 vector, and
+    /// finding its L<sub>1</sub> norm. This is simply the sum of the coefficients + 1
+    /// (the extra 1 comes from the `0 0 1` implicit row).
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use kurbo::Affine;
+    /// # use float_cmp::assert_approx_eq;
+    /// let affine = Affine::new([1., 2., 3., 4., 5., 6.]);
+    /// assert_approx_eq!(f64, affine.norm_l_1_1(), affine.as_coeffs().into_iter().sum::<f64>() + 1.);
+    /// ```
+    ///
+    /// [matrix-norm#entry-wise]: https://en.wikipedia.org/wiki/Matrix_norm#%22Entry-wise%22_matrix_norms
+    pub fn norm_l_1_1(self) -> f64 {
```

Comment:
I added this function for error checking, before deciding that max(element-wise error) was a better way. I've left it in to stimulate conversation about matrix norms, but fully expect to need to remove it before merging.

Reply:
If we did keep it I almost certainly think we should remove the `1`, otherwise |a - a| != 0.